uk.co.compucorp.civicrm.hrleaveandabsences/scss/leaveandabsence.scss

```diff
@@ -0,0 +1,7 @@
+// @file
+// SASS partials import
+// -----------------------------------------------------------------------------
+
+// Leave and absence partials.
+// -----------------------------------------------------------------------------
```

Comment:
There is no need for any of these comments, the whole file should be self-explanatory